### PR TITLE
Feature/d3 asim 3307: Added market summary

### PR DESCRIPTION
**Pull request by spyrostz**

### CODE REVIEW

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: d3a_interface/sim_results/market_summary_info.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from statistics import mean
+from d3a_interface.utils import key_in_dict_and_not_none
+from d3a_interface.sim_results.results_abc import ResultsBaseClass
+from d3a_interface.sim_results import is_trade_external
+
+
+class MarketSummaryInfo(ResultsBaseClass):
+
+    def __init__(self, should_export_plots):
+        self._should_export_plots = should_export_plots
```
Comment (faizan2590) [inline d3a_interface/sim_results/market_summary_info.py:11]: AFAI, its not being used, any reason to introduce for this endpoint?

**Review by faizan2590 — DISMISSED** [review]
One comment, other that LGTM

File: d3a_interface/sim_results/market_summary_info.py (hunk 1 above)
Comment (spyrostz) [reply d3a_interface/sim_results/market_summary_info.py:11]: It is used [here](https://github.com/gridsingularity/d3a-interface/pull/177/files#diff-f98c72ddf2055f5c936524db4d713bec808118a4bc18b4011c97869d6156a09aR34), line 34, in order to not update the endpoint if the plots should be exported

**Pull request merged**